<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainPage</class>
 <widget class="QWidget" name="MainPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1024</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainPage</string>
  </property>
  <widget class="QFrame" name="frame">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>161</width>
     <height>601</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QPushButton" name="pushButton_6">
    <property name="geometry">
     <rect>
      <x>5</x>
      <y>460</y>
      <width>152</width>
      <height>87</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_3">
    <property name="geometry">
     <rect>
      <x>5</x>
      <y>190</y>
      <width>152</width>
      <height>87</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_5">
    <property name="geometry">
     <rect>
      <x>5</x>
      <y>370</y>
      <width>152</width>
      <height>87</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_2">
    <property name="geometry">
     <rect>
      <x>5</x>
      <y>100</y>
      <width>152</width>
      <height>87</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_4">
    <property name="geometry">
     <rect>
      <x>5</x>
      <y>280</y>
      <width>152</width>
      <height>87</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="tempButton">
    <property name="geometry">
     <rect>
      <x>5</x>
      <y>10</y>
      <width>152</width>
      <height>87</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="pushButton_9">
   <property name="geometry">
    <rect>
     <x>620</x>
     <y>540</y>
     <width>161</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(252, 175, 62);
color: rgb(136, 138, 133);
border:0px groove gray;
border-radius:10px;padding:2px 4px;</string>
   </property>
   <property name="text">
    <string>关于我们</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_7">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>540</y>
     <width>161</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(252, 175, 62);
color: rgb(136, 138, 133);
border:0px groove gray;
border-radius:10px;padding:2px 4px;</string>
   </property>
   <property name="text">
    <string>报警设置</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_8">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>540</y>
     <width>161</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(252, 175, 62);
color: rgb(136, 138, 133);
border:0px groove gray;
border-radius:10px;padding:2px 4px;</string>
   </property>
   <property name="text">
    <string>系统测试</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_10">
   <property name="geometry">
    <rect>
     <x>840</x>
     <y>540</y>
     <width>161</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(252, 175, 62);
color: rgb(136, 138, 133);
border:0px groove gray;
border-radius:10px;padding:2px 4px;</string>
   </property>
   <property name="text">
    <string>返回桌面</string>
   </property>
  </widget>
  <widget class="QFrame" name="lampFrame">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>10</y>
     <width>821</width>
     <height>521</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>5</x>
      <y>10</y>
      <width>150</width>
      <height>197</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_11">
    <property name="geometry">
     <rect>
      <x>225</x>
      <y>10</y>
      <width>150</width>
      <height>197</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_12">
    <property name="geometry">
     <rect>
      <x>445</x>
      <y>10</y>
      <width>150</width>
      <height>197</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_13">
    <property name="geometry">
     <rect>
      <x>665</x>
      <y>10</y>
      <width>150</width>
      <height>197</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_14">
    <property name="geometry">
     <rect>
      <x>665</x>
      <y>266</y>
      <width>150</width>
      <height>197</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_15">
    <property name="geometry">
     <rect>
      <x>445</x>
      <y>266</y>
      <width>150</width>
      <height>197</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_16">
    <property name="geometry">
     <rect>
      <x>5</x>
      <y>266</y>
      <width>150</width>
      <height>197</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_17">
    <property name="geometry">
     <rect>
      <x>225</x>
      <y>266</y>
      <width>150</width>
      <height>197</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>210</y>
      <width>141</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>客厅中央灯</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>210</y>
      <width>141</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>客厅灯带</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>210</y>
      <width>141</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>客厅筒灯</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>670</x>
      <y>210</y>
      <width>141</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>餐厅灯</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>670</x>
      <y>470</y>
      <width>141</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>厨房灯</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>470</y>
      <width>141</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>客卧室灯</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>470</y>
      <width>141</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>主卧室灯</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>470</y>
      <width>141</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>卫生间灯</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="lampBackLabel">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>821</width>
      <height>521</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>客厅中央灯</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <zorder>lampBackLabel</zorder>
   <zorder>pushButton</zorder>
   <zorder>pushButton_11</zorder>
   <zorder>pushButton_12</zorder>
   <zorder>pushButton_13</zorder>
   <zorder>pushButton_14</zorder>
   <zorder>pushButton_15</zorder>
   <zorder>pushButton_16</zorder>
   <zorder>pushButton_17</zorder>
   <zorder>label</zorder>
   <zorder>label_2</zorder>
   <zorder>label_3</zorder>
   <zorder>label_4</zorder>
   <zorder>label_5</zorder>
   <zorder>label_6</zorder>
   <zorder>label_7</zorder>
   <zorder>label_8</zorder>
  </widget>
  <widget class="QFrame" name="testFrame">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>10</y>
     <width>821</width>
     <height>521</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QPushButton" name="pushButton_26">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>30</y>
      <width>141</width>
      <height>141</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="testBackLabel">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>821</width>
      <height>521</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <zorder>testBackLabel</zorder>
   <zorder>pushButton_26</zorder>
  </widget>
  <widget class="QFrame" name="tempFrame">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>10</y>
     <width>821</width>
     <height>521</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QProgressBar" name="progressBar">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>50</y>
      <width>351</width>
      <height>25</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="value">
     <number>24</number>
    </property>
    <property name="format">
     <string>%p℃</string>
    </property>
   </widget>
   <widget class="QProgressBar" name="progressBar_2">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>50</y>
      <width>351</width>
      <height>25</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="value">
     <number>24</number>
    </property>
    <property name="format">
     <string>%p℃</string>
    </property>
   </widget>
   <widget class="QProgressBar" name="progressBar_3">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>160</y>
      <width>351</width>
      <height>25</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="value">
     <number>24</number>
    </property>
    <property name="format">
     <string>%p℃</string>
    </property>
   </widget>
   <widget class="QProgressBar" name="progressBar_4">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>160</y>
      <width>351</width>
      <height>25</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="value">
     <number>24</number>
    </property>
    <property name="format">
     <string>%p℃</string>
    </property>
   </widget>
   <widget class="QProgressBar" name="progressBar_5">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>270</y>
      <width>351</width>
      <height>25</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="value">
     <number>24</number>
    </property>
    <property name="format">
     <string>%p℃</string>
    </property>
   </widget>
   <widget class="QProgressBar" name="progressBar_6">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>270</y>
      <width>351</width>
      <height>25</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="value">
     <number>24</number>
    </property>
    <property name="format">
     <string>%p℃</string>
    </property>
   </widget>
   <widget class="QProgressBar" name="progressBar_7">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>370</y>
      <width>351</width>
      <height>25</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="value">
     <number>24</number>
    </property>
    <property name="format">
     <string>%p℃</string>
    </property>
   </widget>
   <widget class="QProgressBar" name="progressBar_8">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>370</y>
      <width>351</width>
      <height>25</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="value">
     <number>24</number>
    </property>
    <property name="format">
     <string>%p℃</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_17">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>90</y>
      <width>241</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>客厅温度</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_18">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>90</y>
      <width>241</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>后阳台温度</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_19">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>210</y>
      <width>241</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>餐厅温度</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_20">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>210</y>
      <width>241</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>前阳台温度</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_21">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>310</y>
      <width>241</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>客卧室温度</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_22">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>310</y>
      <width>241</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>主卧是温度</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_23">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>420</y>
      <width>241</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>厨房温度</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_24">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>420</y>
      <width>241</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>卫生间温度</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="backLabel">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>821</width>
      <height>521</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <zorder>backLabel</zorder>
   <zorder>progressBar</zorder>
   <zorder>progressBar_2</zorder>
   <zorder>progressBar_3</zorder>
   <zorder>progressBar_4</zorder>
   <zorder>progressBar_5</zorder>
   <zorder>progressBar_6</zorder>
   <zorder>progressBar_7</zorder>
   <zorder>progressBar_8</zorder>
   <zorder>label_17</zorder>
   <zorder>label_18</zorder>
   <zorder>label_19</zorder>
   <zorder>label_20</zorder>
   <zorder>label_21</zorder>
   <zorder>label_22</zorder>
   <zorder>label_23</zorder>
   <zorder>label_24</zorder>
  </widget>
  <widget class="QLabel" name="allBackLabel">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1024</width>
     <height>600</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLCDNumber" name="lcdNumber">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>270</y>
     <width>771</width>
     <height>111</height>
    </rect>
   </property>
   <property name="lineWidth">
    <number>0</number>
   </property>
   <property name="digitCount">
    <number>19</number>
   </property>
   <property name="value" stdset="0">
    <double>12345.000000000000000</double>
   </property>
  </widget>
  <zorder>allBackLabel</zorder>
  <zorder>lampFrame</zorder>
  <zorder>frame</zorder>
  <zorder>pushButton_9</zorder>
  <zorder>pushButton_7</zorder>
  <zorder>pushButton_8</zorder>
  <zorder>pushButton_10</zorder>
  <zorder>tempFrame</zorder>
  <zorder>testFrame</zorder>
  <zorder>lcdNumber</zorder>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
